tests/e2e/system/conftest.py

```diff
@@ -0,0 +1,420 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.bucket_utils import (
+    compare_object_checksums_between_bucket_and_local,
+    compare_directory,
+    patch_replication_policy_to_bucket,
+    random_object_round_trip_verification,
+    sync_object_directory,
+    wait_for_cache,
+    write_random_test_objects_to_bucket,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources import pod, pvc
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.helpers.helpers import (
+    wait_for_resource_state,
+    modify_statefulset_replica_count,
+    validate_pv_delete,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_db_backup_and_recovery(request, snapshot_factory):
```

Comment:
can we call and use this https://github.com/red-hat-storage/ocs-ci/blob/5563b4a4bf9bedf3aee7bcdb5e36b79ae98e1b68/tests/e2e/kcs/test_noobaa_db_backup_and_recovery.py#L72 instead of recreating the same function again? Let's try to avoid duplication...

Reply:
we cant call to pytest test (it is not regular function)

Reply:
We need to avoid duplication of code. I suggest we modify the [test_noobaa_db_backup_and_recovery.py](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/kcs/test_noobaa_db_backup_and_recovery.py#L72) such that it can be called from other tests also. Please work with Akarsha on this.

Reply:
Also, please create a github issue to track this work

Reply:
@hnallurv @Akarsha-rai done

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/5119 addresses this